docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -23,6 +23,13 @@ date: "2022-10-11"
 cost: "0"
 ---
 
+<!-- LEARNING GOALS
+After following this tutorial, you will know how to use fake components to prototype if you don't have (the right) components at hand. You will also know how to swap out your fake components for real components when you get them.
```

Comment:
Nit: slightly more straightforward phrasing, avoiding parens:

```suggestion
After following this tutorial, you will know how to use fake components to prototype a project without needing any hardware components. You will also know how to swap out your fake components for real components when you are ready to do so.
```

Goal: 'know how to swap fake for real' is a great addition, we don't cover that anywhere and it is what likely every Mock user will need once succeeding in the tut. 👍